turbopack/crates/turbo-tasks/src/lib.rs

```diff
@@ -303,6 +305,13 @@ pub mod test_helpers {
     pub use super::manager::{current_task_for_testing, with_turbo_tasks_for_testing};
 }
 
+thread_local! {
+    /// The location of the last error that occurred in the current thread.
+    ///
+    /// Used for debugging when errors are sent to telemetry
+    pub static LAST_ERROR_LOCATION: RefCell<Option<String>> = const { RefCell::new(None) };
```

Comment:
Is the `const` block actually needed here?

```suggestion
    pub static LAST_ERROR_LOCATION: RefCell<Option<String>> = RefCell::new(None);
```